src/components/date/navbar/__spec__.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
```

Comment:
please change the name of this file to `navbar.spec.js`

Reply:
changed, thanks